lib/prepare_security.js

```diff
@@ -0,0 +1,119 @@
+import nv from '@pkgjs/nv';
+import auth from './auth.js';
+import Request from './request.js';
+import fs from 'node:fs';
+import path from 'node:path';
+
+export default class SecurityReleaseSteward {
+  constructor(cli) {
+    this.cli = cli;
+  }
+
+  async start() {
+    const { cli } = this;
+    const credentials = await auth({
+      github: true,
+      h1: true
+    });
+
+    const req = new Request(credentials);
+    const create = await cli.prompt(
+      'Create the Next Security Release issue?',
+      { defaultAnswer: true });
+    if (create) {
+      const issue = new SecurityReleaseIssue(req);
+      const content = await issue.buildIssue(cli);
+      const data = await req.createIssue('Next Security Release', content, {
+        owner: 'nodejs-private',
+        repo: 'node-private'
+      });
+      if (data.html_url) {
+        cli.ok('Created: ' + data.html_url);
+      } else {
+        cli.error(data);
+      }
+    }
+  }
+}
+
+class SecurityReleaseIssue {
+  constructor(req) {
+    this.req = req;
+    this.content = '';
+    this.title = 'Next Security Release';
+    this.affectedLines = {};
+  }
+
+  getSecurityIssueTemplate() {
+    return fs.readFileSync(
+      path.resolve(
+        'lib',
+        'github',
+        'templates',
+        'next-security-release.md'
+      )
```

Comment:
Let's not make this dependent on the cwd
```suggestion
      new URL(
        './github/templates/next-security-release.md',
        import.meta.url
      )
```